src/Components/OATPropertyEditor/Utils.ts

```diff
@@ -261,4 +262,7 @@ export const handleIdChange = (
     } else {
         setIdErrorLength(true);
     }
+    if (value.length > 0) {
```

Comment:
Do we need to check if `setIdWarning` is null or not?

This function is used in a few places... are they all updated to pass this new prop?

Reply:
I don't think it will ever be a null since its the callback to set the value of the idWarning state const
![image](https://user-images.githubusercontent.com/97867829/176506722-a221adc9-f0f0-4d5f-8021-df64359eecc9.png)

Reply:
Right - but this function is used by other components too and they don't invoke this with the setIdWarning parameter? See FormAddEnumItem and FormRootModelDetails.

Reply:
You are Right and those doesn't need it. Done